public/scripts/group-chats.js

```diff
@@ -785,7 +786,8 @@ async function generateGroupWrapper(by_auto_mode, type = null, params = {}) {
         sendSystemMessage(system_message_types.EMPTY, '', { isSmallSys: true });
         return Promise.resolve();
     }
-
+    
+    const showChatQueue = (!(typingIndicator.length === 0 && !isStreamingEnabled()) && openGroupId);
```

Comment:
Is there a reason you explicitly excluded streaming from displaying the queue order?
I am not using group chats a lot, so I might be missing something here.
But wouldn't generally having that indicator, even if the first char in the queue is streaming its response right now, still help?

Reply:
Agreed that streaming should not be a problem here since the group draft loop actually waits for the stream to complete before proceeding to the next member. So calling a refresh list at the beginning of every iteration should be possible

Reply:
I was trying to avoid conflicting with the typing indicator (code around line 799)...with the addition of an on/off toggle in latest commit, that concern goes away, so I've removed that code.